node_modules/oae-principals/lib/internal/dao.js

```diff
@@ -82,8 +83,14 @@ var createUser = module.exports.createUser = function(user, callback) {
  */
 var createGroup = module.exports.createGroup = function(groupId, tenantAlias, displayName, description, visibility, joinable, createdBy, callback) {
     var lastModified = Date.now();
+    var created = new Date(lastModified);
+
+    lastModified = lastModified.toString();
+    created = created.toUTCString();
```

Comment:
Hmm, that's annoying, you'd really expect the two to be the same datatype. Is this just the case with `Principals` or with other tables as well?